src/updater.ts

```diff
@@ -66,14 +66,27 @@ export class Updater {
     await this.loadTargets(MetadataKind.Targets, MetadataKind.Root);
   }
 
+  // Returns the TargetFile info for all targets in the trusted set.
+  public async getTargets(): Promise<TargetFile[]> {
+    if (!this.trustedSet.targets) {
+      await this.refresh();
+    }
+
+    if (this.trustedSet.targets?.signed.targets) {
+      return Object.values(this.trustedSet.targets?.signed.targets);
+    }
+
+    return [];
```

Comment:
I think we can reduce this all to just:
```suggestion
return Object.values(this.trustedSet.targets?.signed.targets || {});
```